packages/expo-modules-core/src/uuid/uuid.web.ts

```diff
@@ -0,0 +1,21 @@
+import sha1 from './lib/sha1';
+import v35 from './lib/v35';
+import { Uuid } from './uuid.types';
+type UUID = string;
+
+// In some cases it is necessary to explicitly import the crypto module
+const cryptoObject = typeof crypto === 'undefined' ? require('crypto') : crypto;
+const randomUuid =
+  cryptoObject && cryptoObject.randomUUID && cryptoObject.randomUUID.bind(cryptoObject);
```

Comment:
Better to use optional chaining rather than `&&`.
Binding seems unnecessary, why not just run `cryptoObject.randomUUID()` inside the `uuidv4` function?

Reply:
Without the binding function doesn't work on web (works fine in node though).